README.md

```diff
@@ -43,18 +43,26 @@ cd ../
 catkin_make_isolated --pkg robot_blockly --install
 ```
 
-To start the backend and build in HTTP server:
+To start the backend and build-in HTTP server:
 ```
 source install_isolated/setup.bash 
 roslaunch robot_blockly robot_blockly.launch 
 ```
 
+In order to include blocks library (e.g. [sr_blockly_blocks](https://github.com/shadow-robot/sr_blockly_blocks)) run:
+```
+source install_isolated/setup.bash
+roslaunch robot_blockly robot_blockly.launch block_packages:=[sr_blockly_blocks]
+```
+
 Point your browser to: [http://localhost:8000/pages/blockly.html](http://localhost:8000/pages/blockly.html)
 
 ### Create your own blocks
-- Open [http://localhost:8000/blockly/demos/blockfactory/index.html](http://localhost:8000/blockly/demos/blockfactory/index.html)
-- Design you own block and then add the metadata to: `frontend/blockly/blocks` and `frontend/blockly/generator`
-- Launch `python build.py` to regenerate the blocks.
+- Default blockly blocks are stored in the blockly package. If you require additional block repositories for your project, these repositories should be cloned into your workspace at the same level as robot_blockly. Then rebuild your workspace.
+- If you want to create your own blocks, you can either add them to an existing ros package or create a new package.
+- Blocks can be generated by opening [http://localhost:8000/blockly/demos/blockfactory/index.html](http://localhost:8000/blockly/demos/blockfactory/index.html)
```

Comment:
should this be:  [http://localhost:8000/blockly/demos/blockfactory/index.html](frontend/blockly/demos/blockfactory/index.html)?